generators/app/templates/src/theme/__ionic.theme-variables.scss

```diff
@@ -32,54 +26,110 @@ $app-direction: ltr;
+  --ion-color-tertiary-contrast-rgb: 255, 255, 255;
+  --ion-color-tertiary-shade: #d7953a;
+  --ion-color-tertiary-tint: #f5b255;
+
+  /** success **/
+  --ion-color-success: #10dc60;
+  --ion-color-success-rgb: 16, 220, 96;
+  --ion-color-success-contrast: #fff;
+  --ion-color-success-contrast-rgb: 255, 255, 255;
+  --ion-color-success-shade: #0ec254;
+  --ion-color-success-tint: #28e070;
+
+  /** warning **/
+  --ion-color-warning: #ffce00;
+  --ion-color-warning-rgb: 255, 206, 0;
+  --ion-color-warning-contrast: #000;
+  --ion-color-warning-contrast-rgb: 0, 0, 0;
+  --ion-color-warning-shade: #e0b500;
+  --ion-color-warning-tint: #ffd31a;
+
+  /** danger **/
+  --ion-color-danger: #f53d3d;
+  --ion-color-danger-rgb: 245, 61, 61;
+  --ion-color-danger-contrast: #fff;
+  --ion-color-danger-contrast-rgb: 255, 255, 255;
+  --ion-color-danger-shade: #d83636;
+  --ion-color-danger-tint: #f65050;
+
+  /** light **/
+  --ion-color-light: #f4f4f4;
+  --ion-color-light-rgb: 244, 244, 244;
+  --ion-color-light-contrast: #000;
+  --ion-color-light-contrast-rgb: 0, 0, 0;
+  --ion-color-light-shade: #d7d7d7;
+  --ion-color-light-tint: #f5f5f5;
+
+  /** medium **/
+  --ion-color-medium: #989aa2;
+  --ion-color-medium-rgb: 152, 154, 162;
+  --ion-color-medium-contrast: #000;
+  --ion-color-medium-contrast-rgb: 0, 0, 0;
+  --ion-color-medium-shade: #86888f;
+  --ion-color-medium-tint: #a2a4ab;
+
+  /** dark **/
+  --ion-color-dark: #222;
+  --ion-color-dark-rgb: 34, 34, 34;
+  --ion-color-dark-contrast: #fff;
+  --ion-color-dark-contrast-rgb: 255, 255, 255;
+  --ion-color-dark-shade: #1e1e1e;
+  --ion-color-dark-tint: #383838;
+
+  /** background **/
+  --ion-background-color: #fff;
+}
 
 
 // App iOS Variables
 // --------------------------------------------------
 // iOS only Sass variables can go here
 
-$cordova-ios-statusbar-padding:           20px;
-$text-input-ios-show-invalid-highlight:   true;
-
 
 // App Material Design Variables
 // --------------------------------------------------
 // Material Design only Sass variables can go here
 
-$text-input-md-show-valid-highlight:      false;
-
 
 // App Windows Variables
 // --------------------------------------------------
 // Windows only Sass variables can go here
 
-$text-input-wp-show-valid-highlight:      false;
-
 
 // App Theme
 // --------------------------------------------------
 // Ionic apps can have different themes applied, which can then be future customized. This import comes last so that
 // the above variables are used and Ionic's default are overridden.
 
-@import "~ionic-angular/themes/ionic.theme.default";
-
 
 // Ionicons
 // --------------------------------------------------
 // The premium icon font for Ionic. For more info, please see: http://ionicframework.com/docs/v2/ionicons/
 
-@import "~ionicons/dist/scss/ionicons";
-
 
 // Fonts
 // --------------------------------------------------
 
-@import "~ionic-angular/fonts/roboto";
-@import "~ionic-angular/fonts/noto-sans";
```

Comment:
comments related to the imports should be removed as well, they were from Ionic code 😉

Reply:
removed for now :) I expect them to come back when they expand upon theming.